README.md

```diff
@@ -33,29 +33,124 @@ Role Variables
 - `torrc_hidden_services: []` List of `{dir: '.../hidden_service', ports: ['x y:z']}` dicts to describe hidden services. "x y:z" says to redirect requests on port x to the address y:z. Once you have configured a hidden service, you can look at the contents of the file ".../hidden_service/hostname" for the address to tell people.
 #### Tor relay options
 - `torrc_or_ports: []`
+Required: what port(s) to advertise for incoming Tor connections.
```

Comment:
This comment says that this is required. Do we have any way of enforcing it? 
I'm aware that the answer is "no". We briefly discussed this offline - I would like to know which is the standard way of dealing with this validations.